bin/phpunit-docker.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+WP_VERSION=${1-latest}
+
+MYSQL_ROOT_PASSWORD='wordpress'
+docker network create tests
+db=$(docker run --network tests --name db -e MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD -d mariadb)
+function cleanup() {
+	docker rm -f $db
+	docker network rm tests
+}
+trap cleanup EXIT
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+WP_VERSION=$($DIR/download-wp-tests.sh wordpress_test root "$MYSQL_ROOT_PASSWORD" "db" "$WP_VERSION")
+
+## Ensure there's a database connection for the rest of the steps
+until docker exec -it $db mysql -u root -h db --password="wordpress" -e 'CREATE DATABASE wordpress_test' > /dev/null; do
+	echo "Waiting for database connection..."
+	sleep 5
+done
+
+docker run \
+ 	--network tests \
+	-v $(pwd):/app \
+	-v /tmp/wordpress-tests-lib-$WP_VERSION:/tmp/wordpress-tests-lib \
+	-v /tmp/wordpress-$WP_VERSION:/tmp/wordpress \
+	--rm phpunit/phpunit \
```

Comment:
One minor downside to this image is that it's running PHP 7.0 (https://hub.docker.com/r/phpunit/phpunit/)

Reply:
> One minor downside to this image is that it's running PHP 7.0

I think it depends what version we use. For example, I see the PHP Unit 7.4.0 container runs Alpine 3.8. If I `docker run -it alpine:3.8` and install the `php7` package, it actually installs `PHP 7.2.10`. It looks like there's no consistency for what version of Alpine they're running though. Some of the Dockerfiles say 3.8, some say 3.7, and some say latest.

https://github.com/JulienBreux/phpunit-docker/blob/master/7.4.0/dockerfile#L2